MerlinAU.sh

```diff
@@ -555,7 +602,7 @@ _CheckForNewScriptUpdates_()
    if [ "$DLRepoVersionNum" -gt "$ScriptVersionNum" ]
    then
       scriptUpdateNotify="New script update available.
-${REDct}v$SCRIPT_VERSION${NOct} --> ${GRNct}v$DLRepoVersion${NOct}"
+${REDct}v${SCRIPT_VERSION}${NOct} --> ${GRNct}v$DLRepoVersion${NOct}"
```

Comment:
Identified a little "bug" with the update function here.
Don't think it had any impact, but my editor was detecting this as a variable called "$SCRIPT_VERSION$" instead of "$SCRIPT_VERSION"

Reply:
> Identified a little "bug" with the update function here. Don't think it had any impact, but my editor was detecting this as a variable called "$SCRIPT_VERSION$" with the extra dollar sign at the end, instead of "$SCRIPT_VERSION"
>
>
Good catch!! And you're right; the way it was had no effect on the actual functionality because the shell interpreter parser is smart enough to distinguish the 2 variable references within the expression. However, it's always recommended to clearly & unambiguously separate each variable reference when they are next to each other like that, and it's also considered a best-practice coding standard.

Reply:
Exactly. I know the shell interpreter is smart enough, this is probably a limitation of my editor, but I figured it's an easy cleanup while I was here.